ferrilata-devilline/Services/Helpers/Extensions/DatabaseSeeder.cs

```diff
@@ -57,27 +63,98 @@ private static List<Level> CreateLevels(ApplicationContext context, List<Badge>
             var level1 = new Level
             {
                 LevelNumber = 1,
-                Description = "level1 description",
+                Description = "Receive feedback",
                 Weight = "level1 weight",
                 Badge = badges[0]
             };
+            var level11 = new Level
+            {
+                LevelNumber = 11,
+                Description = "Receive feedback well",
+                Weight = "level11 weight",
+                Badge = badges[0]
+            };
+            var level12 = new Level
+            {
+                LevelNumber = 12,
+                Description = "Receive feedback very well",
+                Weight = "level12 weight",
+                Badge = badges[0]
+            };
+            var level13 = new Level
+            {
+                LevelNumber = 13,
+                Description = "Receive feedback exceptionally well",
+                Weight = "level13 weight",
+                Badge = badges[0]
+            };
             var level2 = new Level
             {
                 LevelNumber = 2,
-                Description = "level2 description",
+                Description = "Speak English",
                 Weight = "level2 weight",
                 Badge = badges[1]
             };
+            var level21 = new Level
+            {
+                LevelNumber = 21,
```

Comment:
the level numbers are a bit random (receives feedback: levelnum 1, receives feedback well: levelnum 11) but it doesnt hurt the functionality so its fine

Reply:
Yeah, I was not too creating, I will fix it in the next polishing iteration